server/src/utils/index.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/prefer-default-export */
+import type { Generator } from './index.types';
+
+/**
+ * Generate an array of casual generator
+ *
+ * @param times - number time to repeat element on array
+ * @param generators - the casual generator and id to repeat
+ * @returns
+ */
+export const arrayOfCasual = (times: number, generators: Generator[]) => {
```

Comment:
what is the usage of this?

Reply:
Like JSDoc description said, it use for generate an array of casual for test, inserted into fake mongo db